app/webpack/stylesheets/openstudios/_container_header.scss

```diff
@@ -0,0 +1,30 @@
+.container-header {
+  border-bottom: 2px solid $gto_gray;
+  width: 100%;
+  padding: 7px $openstudios-main-margin;
+}
+.container-header__brand {
+  float: left;
+  position: absolute;
+  display: flex;
+  align-items: center;
+}
+.container-header__mau-title {
+  padding-left: 5px;
+  color: $gto_gray;
```

Comment:
again - little design direction but let's use got_dark_teal here to give it a tiny bit of spice.